plugin/src/main/java/com/denizenscript/denizen/events/player/PlayerPlacesBlockScriptEvent.java

```diff
@@ -47,18 +48,21 @@ public class PlayerPlacesBlockScriptEvent extends BukkitScriptEvent implements L
     // @Example
     // on player places cactus against:sand:
     //
+    // @Example
+    // on player places block type:cactus:

```

Comment:
should probably have a `!` with a `#` comment to explain it's for any block *except* cactus. (ie so the example shows the actual intended value of the feature)

Reply:
Done.